scrapli/response.py

```diff
@@ -226,10 +224,7 @@ def raise_for_status(self) -> None:
             raise ScrapliCommandFailure()
 
 
-if TYPE_CHECKING:
-    ScrapliMultiResponse = UserList[Response]  # pylint:  disable=E1136; # pragma:  no cover
-else:
-    ScrapliMultiResponse = UserList
+ScrapliMultiResponse = UserList[Response] if TYPE_CHECKING else UserList
```

Comment:
This appears to have angered Mypy! 😆 Its actually kind of good cause it made me re-look at some of the typing things in there and why this upset it. I'm not entirely sure I understand why this line (which is of course functionally the same... at least I think... as the way it was before breaks things, but it did! It also made it so that the `return response.host` below was upset... *that* was actually just bad typing on my part though... here is what I came up with to "fix" this -- let me know what you think:

```
if TYPE_CHECKING:
    ScrapliMultiResponse = UserList[Response]
else:
    ScrapliMultiResponse = UserList


class MultiResponse(ScrapliMultiResponse):
    def __init__(self, initlist: Optional[Iterable[Any]] = None) -> None:
        super().__init__(initlist=initlist)

        self.data: List[Response]
```

Without the if/else mypy complains that the `ScrapliMultiResponse` is not a valid type, so I put that back in testing, but added the init so we can explicitly type the `self.data` as a list of `Response` which solves the other mypy complaints.